solidity/contracts/TokenStaking.sol

```diff
@@ -578,18 +579,16 @@ contract TokenStaking is Authorizations, StakeDelegatable {
         address _operator,
         address _operatorContract
     ) public view returns (uint256 balance) {
-        bool isAuthorized = isAuthorizedForOperator(_operator, _operatorContract);
-
         uint256 operatorParams = operators[_operator].packedParams;
-
-        bool isActive = _isInitialized(operatorParams);
-        // `undelegatedAt` may be set to a time in the future,
-        // to schedule undelegation in advance.
-        // In this case the operator is still eligible
-        // until the timestamp `undelegatedAt`.
-        bool isUndelegating = _isUndelegating(operatorParams);
-
-        if (isAuthorized && isActive && !isUndelegating) {
+        if (
+            isAuthorizedForOperator(_operator, _operatorContract) &&
+            _isInitialized(operatorParams) &&
+            !_isUndelegating(operatorParams)
+            // `undelegatedAt` may be set to a time in the future,
+            // to schedule undelegation in advance.
+            // In this case the operator is still eligible
+            // until the timestamp `undelegatedAt`.
```

Comment:
Thoughts on moving the clarification comment out of the if condition?

Reply:
I am happy you brought it up, I wasn't sure what's the best option here. This comment is only for the line `!_isUndelegating(operatorParams)` and originally I placed it above this line but then, the linter is unhappy about `&&` rules. What's the best solution here in your opinion?

Reply:
Yep, linter does that 😞  I think that placing it above the whole if statement would be better than below the line it relates to, we could rephrase the doc to clarify to what part it relates to.

Reply:
Please see https://github.com/keep-network/keep-core/pull/1902/commits/00a7398bd0f4fd1a4955a120ef9d90d3b60aec6b